### PR TITLE
Revise probability phần 5&6

### LINKED ISSUE #690: Revise "probability_vn" - Phần 5

### LINKED ISSUE #691: Revise "probability_vn" - Phần 6

### PR DESCRIPTION
**Pull request by tiepvupsu**
close #690 
close #691

### CODE REVIEW

File: chapter_preliminaries/probability_vn.md [hunk 1]
```diff
@@ -533,13 +533,14 @@ does not reveal any information about the occurrence of an event of $B$.
 In this case $P(B \mid A) = P(B)$. Statisticians typically express this as $A \perp  B$. From Bayes' theorem, it follows immediately that also $P(A \mid B) = P(A)$.
 In all the other cases we call $A$ and $B$ dependent. For instance, two successive rolls of a die are independent. In contrast, the position of a light switch and the brightness in the room are not (they are not perfectly deterministic, though, since we could always have a broken light bulb, power failure, or a broken switch).
 -->
-Một tính chất hữu ích khác để kiểm tra là *tính phụ thuộc* với *tính độc lập*.
-Hai biến ngẫu nhiên $A$ và $B$ độc lập nghĩa là sự xuất hiện của một sự kiện của $A$ không tiết lộ bất kỳ thông tin nào về sự xuất hiện của một sự kiện của $B$.
-Trong trường hợp này $P(B \mid A) = P(B)$. Các nhà thống kê thường biểu thị điều này là $A \perp B$. 
+Một tính chất hữu ích khác cần kiểm tra là *tính phụ thuộc* với *tính độc lập*.
+Hai biến ngẫu nhiên $A$ và $B$ độc lập nghĩa là việc một sự kiện của $A$ xảy ra không tiết lộ bất kỳ thông tin nào về việc xảy ra một sự kiện của $B$.
+Trong trường hợp này $P(B \mid A) = P(B)$.
+Các nhà thống kê thường biểu diễn việc này bởi $A \perp B$.
```
Comment (minhduc0711) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Các nhà thống kê thường biểu diễn điều này bằng ký hiệu $A \perp B$. 
```

File: chapter_preliminaries/probability_vn.md [hunk 2]
```diff
@@ -548,7 +549,8 @@ if and only if $P(A, B \mid C) = P(A \mid C)P(B \mid C)$. This is expressed as $
 -->
 
 Vì $P(A \mid B) = \frac{P(A, B)}{P(B)} = P(A)$ tương đương với $P(A, B) = P(A)P(B)$, hai biến ngẫu nhiên là độc lập khi và chỉ khi phân phối đồng thời của chúng là tích các phân phối riêng lẻ của chúng.
-Tương tự, cho một biến ngẫu nhiên $C$ khác, hai biến ngẫu nhiên $A$ và $B$ là *độc lập có điều kiện* khi và chỉ khi $P(A, B \mid C) = P(A \mid C)P(B \mid C)$. Điều này được biểu thị bằng $A \perp B \mid C$.
+Tương tự, cho một biến ngẫu nhiên $C$ khác, hai biến ngẫu nhiên $A$ và $B$ là *độc lập có điều kiện* khi và chỉ khi $P(A, B \mid C) = P(A \mid C)P(B \mid C)$.
+Điều này được ký hiệu bởi $A \perp B \mid C$.
```
Comment (minhduc0711) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Điều này được ký hiệu là $A \perp B \mid C$.
```

File: chapter_preliminaries/probability_vn.md [hunk 3]
```diff
@@ -563,24 +565,24 @@ it never fails to detect HIV if the patient actually has it. We use $D_1$ to ind
 :numref:`conditional_prob_D1` lists such conditional probability.
 -->
 
-Hãy thử nghiệm các kiến thức chúng ta vừa học. Giả sử rằng một bác sĩ phụ trách xét nghiệm AIDS cho một bệnh nhân. 
+Hãy thử nghiệm các kiến thức chúng ta vừa học.
+Giả sử rằng một bác sĩ phụ trách xét nghiệm AIDS cho một bệnh nhân. 
 Việc xét nghiệm này khá chính xác và nó chỉ thất bại với xác suất $1\%$ nếu bệnh nhân khỏe mạnh nhưng lại báo cáo nhầm anh ta bị bệnh.
```
Comment (minhduc0711) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Việc xét nghiệm này khá chính xác và nó chỉ thất bại với xác suất $1\%$, khi nó cho kết quả dương tính dù bệnh nhân khỏe mạnh. 
```

File: chapter_preliminaries/probability_vn.md [hunk 4]
```diff
@@ -563,24 +565,24 @@ it never fails to detect HIV if the patient actually has it. We use $D_1$ to ind
 :numref:`conditional_prob_D1` lists such conditional probability.
 -->
 
-Hãy thử nghiệm các kiến thức chúng ta vừa học. Giả sử rằng một bác sĩ phụ trách xét nghiệm AIDS cho một bệnh nhân. 
+Hãy thử nghiệm các kiến thức chúng ta vừa học.
+Giả sử rằng một bác sĩ phụ trách xét nghiệm AIDS cho một bệnh nhân. 
 Việc xét nghiệm này khá chính xác và nó chỉ thất bại với xác suất $1\%$ nếu bệnh nhân khỏe mạnh nhưng lại báo cáo nhầm anh ta bị bệnh. 
 Hơn thế nữa, nó không bao giờ thất bại trong việc phát hiện HIV nếu bệnh nhân thực sự nhiễm.
```
Comment (minhduc0711) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Hơn nữa, nó không bao giờ thất bại trong việc phát hiện HIV nếu bệnh nhân thực sự bị nhiễm bệnh. 
```

File: chapter_preliminaries/probability_vn.md [hunk 5]
```diff
@@ -563,24 +565,24 @@ it never fails to detect HIV if the patient actually has it. We use $D_1$ to ind
 :numref:`conditional_prob_D1` lists such conditional probability.
 -->
 
-Hãy thử nghiệm các kiến thức chúng ta vừa học. Giả sử rằng một bác sĩ phụ trách xét nghiệm AIDS cho một bệnh nhân. 
+Hãy thử nghiệm các kiến thức chúng ta vừa học.
+Giả sử rằng một bác sĩ phụ trách xét nghiệm AIDS cho một bệnh nhân. 
 Việc xét nghiệm này khá chính xác và nó chỉ thất bại với xác suất $1\%$ nếu bệnh nhân khỏe mạnh nhưng lại báo cáo nhầm anh ta bị bệnh. 
 Hơn thế nữa, nó không bao giờ thất bại trong việc phát hiện HIV nếu bệnh nhân thực sự nhiễm. 
 Ta sử dụng $D_1$ để chỉ định chẩn đoán ($1$ nếu dương tính và $0$ nếu âm tính) và $H$ để biểu thị tình trạng nhiễm HIV ($1$ nếu dương tính và $0$ nếu âm tính).
```
Comment (minhduc0711) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Ta sử dụng $D_1$ để biểu diễn kết quả chẩn đoán ($1$ nếu dương tính và $0$ nếu âm tính) và $H$ để biểu thị tình trạng nhiễm HIV ($1$ nếu dương tính và $0$ nếu âm tính).
```

File: chapter_preliminaries/probability_vn.md [hunk 6]
```diff
@@ -592,9 +594,9 @@ Ta sử dụng $D_1$ để chỉ định chẩn đoán ($1$ nếu dương tính
 Note that the column sums are all $1$ (but the row sums are not), since the conditional probability needs to sum up to $1$, just like the probability. Let's work out the probability of the patient having AIDS if the test comes back positive, i.e., $P(H = 1 \mid D_1 = 1)$. Obviously this is going to depend on how common the disease is, since it affects the number of false alarms. Assume that the population is quite healthy, e.g., $P(H=1) = 0.0015$. To apply Bayes' Theorem, we need to apply marginalization and the multiplication rule to determine
 -->
 
-Lưu ý rằng tổng của từng cột đều bằng $1$ (nhưng tổng từng hàng thì không), vì xác suất có điều kiện cần cộng lên đến $1$, giống như xác suất.
-Chúng ta hãy tìm ra xác suất bệnh nhân bị AIDS nếu xét nghiệm trở thành dương tính, tức là, $P(H = 1 \mid D_1 = 1)$.
-Rõ ràng điều này sẽ phụ thuộc vào mức độ phổ biến của bệnh, bởi vì nó ảnh hưởng đến số lượng báo động sai.
+Lưu ý rằng tổng của từng cột đều bằng $1$ (nhưng tổng từng hàng thì không), vì xác suất có điều kiện cần có tổng bằng $1$, giống như xác suất.
+Chúng ta hãy tìm ra xác suất bệnh nhân bị AIDS nếu xét nghiệm trả về kết quả dương tính, tức là, $P(H = 1 \mid D_1 = 1)$.
```
Comment (minhduc0711) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Hãy cùng tìm xác suất bệnh nhân bị AIDS nếu xét nghiệm trả về kết quả dương tính, tức $P(H = 1 \mid D_1 = 1)$.
```

File: chapter_preliminaries/probability_vn.md [hunk 7]
```diff
@@ -831,27 +831,14 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
...
-<!-- Phần 2 -->
 * Lê Khắc Hồng Phúc
 * Phạm Hồng Vinh
 * Mai Sơn Hải
-
-<!-- Phần 4 -->
 * Trần Kiến An
-
-<!-- Phần 5 -->
 * Tạ H. Duy Nguyên
```
Comment (tiepvupsu) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
* Tạ H. Duy Nguyên
* Phạm Minh Đức
```

File: chapter_preliminaries/probability_vn.md [hunk 8]
```diff
@@ -287,7 +287,7 @@ we can say that the event "seeing an odd number" has occurred.
 
 Khi ta thực hiện gieo một con xúc sắc, chúng ta gọi tập hợp $\mathcal{S} = \{1, 2, 3, 4, 5, 6\}$ là *không gian mẫu* hoặc *không gian kết quả*, trong đó mỗi phần tử sẻ là một *kết quả*.
 *Biến cố* là một tập hợp các kết quả của không gian mẫu. 
-Ví dụ, "gieo được một số $5$" ($\{5\}$) và "gieo được một số lẻ" ($\{1, 3, 5\}$) đều là những biến cố hợp lệ khi gieo một con xúc sắc. 
+Ví dụ, "gieo được một số $5$" ($\{5\}$) và "gieo được một số lẻ" ($\{1, 3, 5\}$) đều là những sự kiện hợp lệ khi gieo một con xúc sắc.
```
[the remark this answers is not in the document]
Comment (tiepvupsu) [reply chapter_preliminaries/probability_vn.md]: Oạch, search and replace.

Comment (tiepvupsu) [inline chapter_preliminaries/probability_vn.md]:
```suggestion
Ví dụ, "gieo được một số $5$" ($\{5\}$) và "gieo được một số lẻ" ($\{1, 3, 5\}$) đều là những biến cố hợp lệ khi gieo một con xúc sắc. 
```

**Comment by tiepvupsu** [comment]
cc @minhduc0711 https://github.com/aivivn/d2l-vn/pull/707/commits/335266a53b08f015c0534b69a01a878d58dc0d0c

**Pull request merged**